ntbs-service/Models/Entities/MDRDetails.cs

```diff
@@ -13,6 +14,25 @@ namespace ntbs_service.Models.Entities
     [Display(Name = "MDR details")]
     public partial class MDRDetails : ModelBase, IOwnedEntityForAuditing
     {
+        [NotMapped]
+        public TreatmentRegimen? Treatment { get; set; }
+        public bool IsMDRTreatment => Treatment == Enums.TreatmentRegimen.MdrTreatment;
```

Comment:
This is bad - this is the very validation that we moved these fields to avoid. In particular, the `TreatmentRegimen` in `ClinicalDetails` is actually the "Planned Treatment Regimen" and it should be set soon after the case is first notified. If the patient later moves onto MDR treatment then that field should not be update, but an MDR treatment start date should be set here.

Reply:
Yep I see now after rereading the ticket. Have removed this piece of validation.